src/macros.js

```diff
@@ -86,6 +86,9 @@ defineMacro("\u211A", "\\mathbb{Q}");
 defineMacro("\u211D", "\\mathbb{R}");
 defineMacro("\u2124", "\\mathbb{Z}");
 
+// Unicode middle dot
+defineMacro("\u00b7", "\\cdotp");
```

Comment:
This ends up mapping `\u00b7` to `\cdotp` which maps to `\u22c5`.  But they look the same so that's alright... really the only difference is the bin/punct difference.